src/components/ExtractingFormOcrControl.vue

```diff
@@ -18,7 +18,8 @@ export default {
   data() {
     return {
       textLanguages: [],
-      ocrLanguages: []
+      ocrLanguages: [],
+      hasTesseract: true
```

Comment:
This will always be true, no ?

Reply:
Nop, hasTesseract is set to false when getting the method who gets the languages fails :)